pkg/agent/route/route_linux.go

```diff
@@ -1405,50 +1405,46 @@ func (c *Client) addVirtualNodePortDNATIPRoute(isIPv6 bool) error {
 	return nil
 }
 
-// AddLoadBalancer is used to add routing entry which is used to route LoadBalancer ingress IP to Antrea
-// gateway on host.
-func (c *Client) AddLoadBalancer(svcIP net.IP) error {
-	linkIndex := c.nodeConfig.GatewayConfig.LinkIndex
-	isIPv6 := utilnet.IsIPv6(svcIP)
-	var gw net.IP
-	var mask int
-	if !isIPv6 {
-		gw = config.VirtualServiceIPv4
-		mask = net.IPv4len * 8
-	} else {
-		gw = config.VirtualServiceIPv6
-		mask = net.IPv6len * 8
-	}
-
-	route := generateRoute(svcIP, mask, gw, linkIndex, netlink.SCOPE_UNIVERSE)
-	if err := c.netlink.RouteReplace(route); err != nil {
-		return fmt.Errorf("failed to install route for LoadBalancer ingress IP %s: %w", svcIP.String(), err)
+// AddLoadBalancerIngressIPRoute adds a route entry that forwards traffic to a LoadBalancer ingress IP to the Antrea
+// gateway interface.
+func (c *Client) AddLoadBalancerIngressIPRoute(svcIP net.IP) error {
+	svcIPStr := svcIP.String()
+	if err := c.addServiceRoute(svcIP); err != nil {
+		return fmt.Errorf("failed to install route for LoadBalancer ingress IP %s: %w", svcIPStr, err)
 	}
-	klog.V(4).InfoS("Added LoadBalancer ingress IP route", "route", route)
-	c.serviceRoutes.Store(svcIP.String(), route)
+	klog.V(4).InfoS("Added route for LoadBalancer ingress IP", "IP", svcIPStr)
 
 	return nil
 }
 
-// DeleteLoadBalancer is used to delete routing entry which is used to route LoadBalancer ingress IP to Antrea
-// gateway on host.
-func (c *Client) DeleteLoadBalancer(svcIP net.IP) error {
+// DeleteLoadBalancerIngressIPRoute deletes a route entry that forwards traffic to a LoadBalancer ingress IP to the Antrea
```

Comment:
I think you can simply say "deletes the route entry for the load balancer IP.", rather than repeating the whole description again.

Reply:
Thanks, updated as you suggested.